integration-tests/test-cases/scenario-tests/src/test/java/io/cellery/integration/scenario/tests/BaseTestCase.java

```diff
@@ -85,24 +85,21 @@ protected String run(String orgName, String imageName, String version, String in
         return instanceName;
     }
 
-    protected String run(String orgName, String imageName, String version, String instanceName,
-                                       String[] links, boolean startDependencies, int timeoutSec)
+    protected void run(String orgName, String imageName, String version, String instanceName,
```

Comment:
Why the start with dependencies is removed from this method? Isn't that functionality is used any scenarios? I think in the employee portal, we should be able to start it easily all depended cells with this property.

Reply:
Updated and restored start with dependencies from run function.